diff --git a/dist/shared/client.CRfRve1v.mjs b/dist/shared/client.CRfRve1v.mjs
index 35ed8408c45d128f07f6335c8b00ad9c1781b142..19cdb2e2dc47e484c687787e8609e29096d9b7d9 100644
--- a/dist/shared/client.CRfRve1v.mjs
+++ b/dist/shared/client.CRfRve1v.mjs
@@ -103,7 +103,7 @@ class StandardRPCJsonSerializer {
         return result;
       }
     }
-    if (data instanceof Blob) {
+    if (data instanceof Blob || (typeof data === 'object' && 'uri' in data)) {
       maps.push(segments);
       blobs.push(data);
       return [data, meta, maps, blobs];
